src/features/Layout/Navigation/TopNav.tsx

```diff
@@ -19,7 +19,7 @@ export const TopNav = ({ current, isMobile }: { current: string; isMobile: boole
       justifyContent="flex-end"
       ml={SPACING.BASE}
       mb={SPACING.BASE}
-      mt={SPACING.MD}
+      mt={SPACING.LG}
```

Comment:
?

Reply:
makes it so that the demo mode indicator doesn't cover the icons in tablet view